lib/sendAPN.js

```diff
@@ -77,7 +77,8 @@ class APN {
       collapseId: data.collapseKey,
       mutableContent: data.mutableContent || 0,
       threadId: data.threadId,
-      pushType: data.pushType
+      pushType: data.pushType,
```

Comment:
I checked that the @parse/node-apn version 5.3.2 (the one that is used to send the Apple notification), supports the `interruptionLevel` property.